lib/action_kit/client.rb

```diff
@@ -0,0 +1,17 @@
+module ActionKit
```

Comment:
OK. Can you make a chore to migrate this functionality to the gem and another one for moving the query to the gem's API?